### DISCUSSION TITLE
"yarn install" steps fails intermittent in Github Action CI pipeline

### DISCUSSION THREAD

**Discussion by ram2104** [yarnpkg/berry#5722] [Q&A]
We're using yarn berry exact version "**3.6.3**" in the project. Our CI pipeline runs in the GitHub Actions. But from the past few months, we've observed that the CI pipelines fail intermittently at the `yarn install` step. After trying multiple retries in the GitHub actions, the same step will pass. 

It simply says, `Process Completed with exit code 1` whenever it fails. I need help understanding why it is failing intermittently.

### Attached screenshot. 

- yarn config file

<img width="514" alt="Screenshot 2023-09-08 at 9 28 47 AM" src="https://github.com/yarnpkg/berry/assets/19301579/329cb1cc-4a0e-46b8-b177-13420c294eeb">

- Build log

<img width="843" alt="Screenshot 2023-09-08 at 9 26 00 AM" src="https://github.com/yarnpkg/berry/assets/19301579/782cfcf4-6382-4a18-b588-4ef9b7dd9911">

<img width="453" alt="Screenshot 2023-09-08 at 9 25 52 AM" src="https://github.com/yarnpkg/berry/assets/19301579/fae7fadb-10a2-4dc2-aa46-251cc8f8c927">
.

**Comment by sarahsporck** [comment]
This could be related to https://github.com/yarnpkg/berry/issues/3996

**Comment by arcanis** [comment]
You'll need to look at the build logs. They should be enabled by default on CI I think, but if not: [`enableInlineBuilds`](https://yarnpkg.com/configuration/yarnrc#enableInlineBuilds)
